app/src/utils/utils.ts

```diff
@@ -443,3 +449,29 @@ export async function getCoingeckoDualPrice(): Promise<number> {
   const price = values.usd;
   return price;
 }
+
+export async function getDualHolders(limit = 50, offset = 0): Promise<any> {
+  const url = `https://public-api.solscan.io/token/holders?tokenAddress=DUALa4FC2yREwZ59PHeu1un4wis36vHRv5hWVBmzykCJ&limit=${limit}&offset=${offset}`;
```

Comment:
We prefer pulling the data directly on-chain, rather than depending on solscan's API. Use provider & hlper fcts like findProgramAddressWithMint & getMultipleTokenAccounts. Treasury.tsx has some good examples of how to do this